components/Table.tsx

```diff
@@ -51,20 +52,23 @@ const Name: React.FC<NameProps> = ({ synth }) => {
   const formattedUrl = logo?.fields.file.url
     ? formatContentfulUrl(logo.fields.file.url)
     : CATEGORIES_PLACEHOLDERS[category];
+  const heading = useMemo(() => {
+    if (synth.name) {
+      return synth.name;
+    } else if (synth.type === "emp") {
+      return synth.tokenName;
+    } else {
+      return formatLSPName(synth.longTokenName);
+    }
+  }, [synth]);
```

Comment:
`name` is a new optional field in Contentful CMS. It has the highest priority when displaying the name in the table row